daml-lf/spec/daml-lf-1.rst

```diff
@@ -3677,6 +3787,120 @@ Numeric functions
+
+* ``'ROUNDING_HALF_EVEN' : RoundingMode``
+
+  Round towards the nearest neighbor unless both neighbors are equidistant, in which case round towards the even neighbor.
+
+* ``'ROUNDING_HALF_UP' : RoundingMode``
+
+  Round towards the nearest neighbor unless both neighbors are equidistant, in which case round away from zero.
+
+* ``'ROUNDING_UNNECESSARY' : RoundingMode``
+
+  Throw if the exact result cannot be represented.
+
+BigDecimal functions
+~~~~~~~~~~~~~~~~~~~~
+
+All operations behave as if they first calculate an exact
+math intermediate result and then round this result as specified by
+the ``MathContext``. For an ``UNLIMITED`` math context, all operations
+are exact. If the result cannot be resulted as a ``BigDecimal`` they
+throw an ``ArithmeticError``
+(TODO: https://github.com/digital-asset/daml/issues/8020). For a
+``MATHCTX_BOUNDED roundingMode precision`` math context, the result is exact if it can
+be represented at with at most ``precision`` digits at some scale. If
+the result cannot be represented exactly in at most ``precision``
+digits, the ``precision`` digits in the result are selected according
+to ``roundingMode``.
+
+* ``ADD_BIGDECIMAL : 'MathContext' → 'BigDecimal' → 'BigDecimal'  → 'BigDecimal'``
+
+  Adds the two decimals by the second one rounding the result
+  according to the given ``MathContext``.
+
+* ``SUB_BIGDECIMAL : 'MathContext' → 'BigDecimal' → 'BigDecimal' → 'BigDecimal'``
+
+  Subtracts the second big decimal from the first one by the second one rounding the result
+  according to the given ``MathContext``.
+
+* ``MUL_BIGDECIMAL : 'MathContext' → 'BigDecimal' → 'BigDecimal' → 'BigDecimal'``
+
+  Multiplies the two numerics by the second one rounding the result
+  according to the given ``MathContext``.
+
+* ``DIV_BIGDECIMAL : 'MathContext' → 'BigDecimal' → 'BigDecimal' → 'BigDecimal'``
+
+  Divides the first big decimal by the second one rounding the result
+  according to the given ``MathContext``.
+
+* ``ROUND_BIGDECIMAL : 'MathContext' -> 'BigDecimal' -> 'BigDecimal'``
+
+  Round the big decimal according to the given ``MathContext``.
+
+* ``TO_TEXT_BIGDECIMAL : 'BigDecimal' → 'Text'``
+
+  Returns the numeric string representation of the bigdecimal. The result
+  will be returned at the smallest precision that can represent the result exactly, i.e.,
+  without any trailing zeroes.
+
+* ``FROM_TEXT_BIGDECIMAL : 'Text' → 'Optional' 'BigDecimal'``
+
+  Given a string representation of a numeric returns the numeric
+  wrapped in ``Some``. If the input does not match the regexp
+  ``[+-]?\d+(\.d+)?`` or if the result of the conversion cannot
+  be mapped into a ``BigDecimal`` without loss of precision, returns
+  ``None``.
+
+* ``TO_TEXT_BIGDECIMAL : 'BigDecimal' → 'Text'``
+
+  Returns the numeric string representation of the bigdecimal. The result
+  will be returned at the smallest precision that can represent the result exactly, i.e.,
+  without any trailing zeroes.
```

Comment:
This looks like a repeat.

Reply:
It absolutely is, good catch, 78f88bd4b80ac1a6228af1f65682b168346dc002